python/amici/sbml_import.py

```diff
@@ -468,7 +468,7 @@ def check_event_support(self) -> None:
 
             # check trigger
             trigger_sbml = event.getTrigger()
-            if trigger_sbml.getMath() is None:
+            if trigger_sbml is None or trigger_sbml.getMath() is None:
                 logger.warning(f'Event {event_id} trigger has no trigger '
                                'expression. Ignoring event...')
```

Comment:
```suggestion
                               'expression, so a dummy trigger will be set.')
```

I guess this is now the case?